tests/foreman/cli/test_operatingsystem.py

```diff
@@ -373,3 +382,33 @@ def test_positive_update_parameters_attributes(self):
         os = OperatingSys.info({'id': os_id}, output_format='json')
         self.assertEqual(param_name, os['parameters'][0]['name'])
         self.assertEqual(param_value, os['parameters'][0]['value'])
+
+    @tier4
+    @skip_if_bug_open('bugzilla', 1649011)
+    def test_positive_os_list_with_default_organization_set(self):
+        """list operating systems when the default organization is set
+
+        :id: 2c1ba416-a5d5-4031-b154-54794569a85b
+
+        :BZ: 1649011
+
+        :expectedresults: os list should list operating systems when the
+            default organization is set
+        """
+        make_os()
+        os_list_before_default = OperatingSys.list()
+        self.assertTrue(len(os_list_before_default) > 0)
+        try:
+            Defaults.add({
+                u'param-name': 'organization',
+                u'param-value': DEFAULT_ORG,
+            })
+            result = ssh.command('hammer defaults list')
+            self.assertTrue(DEFAULT_ORG not in "".join(result.stdout))
```

Comment:
Can you paste results? At this step, DEFAULT_ORG should be in defaults. am I missing something?